src/transform.ts

```diff
@@ -240,8 +240,8 @@ export interface ImputeTransform extends ImputeParams {
    */
   key: string;
   /**
-   * An optional array of key values that should be considered for imputation.
-   * If provided, this array will be used in addition to the key values observed within the input data.
+   * An optional array or `ImputeSequence` of key values that should be considered for imputation.
```

Comment:
`ImputeSequence` is an interface in TypeScript, but is not a part of our syntax.  
Plus, just giving the interface name doesn't explain to the users what it is. 

Useful earlier note: 

> Better explain that this "can be an object that define a sequence" and describe how such object looks like.

Reply:
```
Defines the key values that should be considered for imputation.
One: an array of key values or an object defining a [number sequence](https://vega.github.io/vega-lite/docs/impute.html#sequence-def).

If provided, this will be used in addition to the key values observed within the input data.  If not provided, the values will be derived from all unique values of the `key` field. For `impute` in `encoding`, the key field is `x` if `y` field is imputed, or vice versa. 

If there is no impute grouping, this property _must_ be specified. 
```